fhir-notification/src/main/java/com/ibm/fhir/notification/FHIRNotificationService.java

```diff
@@ -34,27 +34,27 @@
     private FHIRNotificationService() {
         log.entering(this.getClass().getName(), "FHIRNotificationService");
         try {
-        // Register the notification service as an interceptor so we can rely on the
-        // interceptor methods to trigger the 'publish' of the notification events.
-        FHIRPersistenceInterceptorMgr.getInstance().addPrioritizedInterceptor(this);
-        initNotificationResourceTypes();
+            // Register the notification service as an interceptor so we can rely on the
+            // interceptor methods to trigger the 'publish' of the notification events.
+            FHIRPersistenceInterceptorMgr.getInstance().addPrioritizedInterceptor(this);
+
+            initNotificationResourceTypes();
         } catch (Throwable t) {
             throw new RuntimeException("Unexpected error during initialization.", t);
         }
         log.exiting(this.getClass().getName(), "FHIRNotificationService");
     }
 
     private void initNotificationResourceTypes() throws Exception {
-        Set<String> includedResourceTypes = Collections.synchronizedSortedSet(new TreeSet<String>());
         List<String> types = FHIRConfiguration.getInstance().loadConfiguration().getStringListProperty(FHIRConfiguration.PROPERTY_NOTIFICATION_RESOURCE_TYPES);
         if (types != null) {
             for (String type : types) {
                 includedResourceTypes.add(type);
             }
         }
 
-        log.finer("Notification service will publish events for these resource types: "
-                + (includedResourceTypes.isEmpty() ? "ALL" : "\n" + includedResourceTypes.toString()));
+        log.info("Notification service when configured will publish events for these resource types: '"
```

Comment:
mind explaining the "when configured" addition?
it reads the config list just above, so isn't it just logging what is configured already?

Reply:
It was set to publish for the whole system, separate from actual enablement.

Reply:
has that been fixed now or that is still coming in a different PR?

Reply:
same PR, since this is executed one time, I left the logging as info

Reply:
```suggestion
        log.info("Notification service, when enabled, will publish events for these resource types: '"
```